app-text/zathura/zathura-0.5.11.ebuild

```
@@ -1,44 +1,43 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit meson virtualx xdg
 
-DESCRIPTION="A highly customizable and functional document viewer"
+DESCRIPTION="Highly customizable & functional document viewer"
 HOMEPAGE="https://pwmt.org/projects/zathura/"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
 	EGIT_BRANCH="develop"
 else
-	SRC_URI="
-		https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz
-	"
+	SRC_URI="https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86 ~amd64-linux ~x86-linux"
 fi
 
 LICENSE="ZLIB"
-SLOT="0/5.6"
-IUSE="man seccomp synctex test"
-
+SLOT="0/6.7" # plugin versions api.abi (see meson.build)
+IUSE="+man seccomp synctex test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
 	dev-libs/json-glib
-	man? ( dev-python/sphinx )
-	seccomp? ( sys-libs/libseccomp )
-	synctex? ( app-text/texlive-core )
 	sys-apps/file
 	x11-libs/cairo
+	x11-libs/pango
 	>=dev-db/sqlite-3.6.23:3
-	>=dev-libs/girara-0.4.3:=
+	>=dev-libs/girara-0.4.5:=
 	>=dev-libs/glib-2.72:2
 	>=x11-libs/gtk+-3.24:3
+	man? ( dev-python/sphinx )
+	seccomp? ( sys-libs/libseccomp )
+	synctex? ( app-text/texlive-core )
 "
 DEPEND="
 	${RDEPEND}
+	>=sys-kernel/linux-headers-5.13
 	test? (
```

```suggestion
	test? (
		dev-libs/appstream-glib
```

https://github.com/pwmt/zathura/blob/9c9380e57d83eb3638348710965fc896f7f71709/data/meson.build#L47